srcts/extras/busy-indicators/busy-indicators.scss

```diff
@@ -0,0 +1,137 @@
+/* This data atttribute is set by ui.busy_indicators.use() */
+[data-shiny-busy-spinners] {
+
+  .recalculating {
+    position: relative;
```

Comment:
> I think the only other possible `position` value that wouldn't break the spinners positioning would be `sticky`?

I thought you were using `position: relative` here so that the `.recalculating` element becomes the anchor for the `::after` pseudo-element. In which case, `position: static` is the `position` that breaks spinner positioning. It's also the default and a very unlikely thing for someone to set on an element using a CSS rule.

What I'm saying is that this rule risks breaking other people's layouts when they've specifically applied `position: xxx` to an output element. There are a few ways this happens, but my example in #4049 demonstrates the problem. In this branch, this rule causes major layout reflowing because the plot outputs in the cards have `position: absolute` that's undone by this rule.